### ISSUE TITLE
Buycraft example not functional

### ISSUE THREAD

**Issue by Katelyn-SN** [meiskam/phpBuycraftAPI#1]
http://prntscr.com/63130t 
After attempting to configure this correctly its invisible on our home page the standalone example however is working correctly.

**Comment by meiskam** [comment]
I made that example a long time ago, and haven't touched it since .. That's the first addon I've made for XenForo/XenPorta, and don't plan on ever making another, because it was so tedious and there's shit for documentation.

You've got XenPorta installed, have configured BuycraftDonateDisplay at "/admin.php?ewrporta/blocks/BuycraftDonateDisplay/options", and added it to your portal at "/admin.php?ewrporta/layouts/portal/edit"?

What versions are you running? I've got it working on:
XenForo 1.2.1
[8wayRun.Com] XenPorta (Portal) 1.6.0
BuycraftDonateDisplay Donation slider block for buycraft (v1.0.2)

I kinda just made this for my own personal use, but if you want to set up an account for me on your forum I could log in and try to see why it's not working.  BTW: where did you find this project from?
